### ISSUE TITLE
Support for copying error results to the clipboard.

### ISSUE THREAD

**Issue by pq** [dart-atom/dartlang#612]
It would be really handy if there was a shortcut to copy error details to the clipboard for pasting into bug reports....

![image](https://cloud.githubusercontent.com/assets/67586/11567931/75915598-999e-11e5-8c8f-cc886d0fae77.png)

(Fun to imagine what iTunes would try and do with strong-mode results! ;))

**Comment by devoncarew** [comment]
Makes sense. For reference though, you can cmd-c the selected text out of the errors view.

**Comment by devoncarew** [comment]
Or perhaps a rich tooltip? I remember mylyn had a really useful tooltip - that could be a place to put actions, like quickfix and copy.

**Comment by pq** [comment]
:+1:   I was able to copy the contents by selecting but it was a bit fussy.  As for mylyn, yeah, that was great and would be awesome here.
